src/wp-includes/fonts/class-wp-font-face.php

```diff
@@ -0,0 +1,400 @@
+		$attributes = '';
+		foreach ( $this->style_tag_attrs as $name => $value ) {
+			$attributes .= " {$name}='{$value}'";
+		}
+		return $attributes;
+	}
+
+	/**
+	 * Gets the `@font-face` CSS styles for locally-hosted font files.
+	 *
+	 * This method does the following processing tasks:
+	 *    1. Orchestrates an optimized `src` (with format) for browser support.
+	 *    2. Generates the `@font-face` for all its fonts.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $font_faces The font-faces to generate @font-face CSS styles.
+	 * @return string The `@font-face` CSS styles.
+	 */
+	private function get_css( $font_faces ) {
+		$css = '';
+
+		foreach ( $font_faces as $font_face ) {
+				// Order the font's `src` items to optimize for browser support.
+				$font_face = $this->order_src( $font_face );
+
+				// Build the @font-face CSS for this font.
+				$css .= '@font-face{' . $this->build_font_face_css( $font_face ) . '}' . "\n";
+		}
+
+		// Don't print the last newline character.
+		return rtrim( $css, "\n" );
+	}
+
+	/**
+	 * Orders `src` items to optimize for browser support.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $font_face Font face to process.
+	 * @return array Font-face with ordered src items.
+	 */
+	private function order_src( array $font_face ) {
+		if ( ! is_array( $font_face['src'] ) ) {
+			$font_face['src'] = (array) $font_face['src'];
+		}
+
+		$src         = array();
+		$src_ordered = array();
+
+		foreach ( $font_face['src'] as $url ) {
+			// Add data URIs first.
+			if ( str_starts_with( trim( $url ), 'data:' ) ) {
+				$src_ordered[] = array(
+					'url'    => $url,
+					'format' => 'data',
+				);
+				continue;
+			}
+			$format         = pathinfo( $url, PATHINFO_EXTENSION );
+			$src[ $format ] = $url;
+		}
+
+		// Add woff2.
+		if ( ! empty( $src['woff2'] ) ) {
+			$src_ordered[] = array(
+				'url'    => $src['woff2'],
+				'format' => 'woff2',
+			);
+		}
+
+		// Add woff.
+		if ( ! empty( $src['woff'] ) ) {
+			$src_ordered[] = array(
+				'url'    => $src['woff'],
+				'format' => 'woff',
+			);
+		}
+
+		// Add ttf.
+		if ( ! empty( $src['ttf'] ) ) {
+			$src_ordered[] = array(
+				'url'    => $src['ttf'],
+				'format' => 'truetype',
+			);
+		}
+
+		// Add eot.
+		if ( ! empty( $src['eot'] ) ) {
+			$src_ordered[] = array(
+				'url'    => $src['eot'],
+				'format' => 'embedded-opentype',
+			);
+		}
+
+		// Add otf.
+		if ( ! empty( $src['otf'] ) ) {
+			$src_ordered[] = array(
+				'url'    => $src['otf'],
+				'format' => 'opentype',
+			);
+		}
+		$font_face['src'] = $src_ordered;
+
+		return $font_face;
+	}
+
+	/**
+	 * Builds the font-family's CSS.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $font_face Font face to process.
+	 * @return string This font-family's CSS.
+	 */
+	private function build_font_face_css( array $font_face ) {
+		$css = '';
+
+		// Wrap font-family in quotes if it contains spaces
+		// and is not already wrapped in quotes.
```

Comment:
```suggestion
		/*
		 * Wrap font-family in quotes if it contains spaces
		 * and is not already wrapped in quotes.
		 */
```
See [the handbook entry](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments) for how Core formats multi-line comments.

Reply:
Thank you @costdev! Done in https://github.com/WordPress/wordpress-develop/pull/5051/commits/4a1d4c2b55ee54cb8563ad06ed9237ce5f73ab6a ✅